azure-pipelines-dependency-graph-testing.yml

```diff
@@ -261,7 +323,7 @@ stages:
               versionSpec: '$(python.version)'
             displayName: 'Use Python $(python.version)'
 
-          - bash: python -m pip install --upgrade pip==20.2.4
+          - bash: python -m pip install --upgrade pip==21.3.1
```

Comment:
Per the pip fiasco we recently dealt with, all stages use `21.3.1` except for usage stats.